config.go

```diff
@@ -108,6 +108,15 @@ const (
 	defaultDiskBackoff  = time.Minute
 	defaultDiskAttempts = 0
 
+	// Set defaults for a health check which ensures that the TLS certificate
+	// is not expired. Although this check is off by default (not all setups
+	// require it), we still set the other default values so that the health
+	// check can be easily enabled with sane defaults.
+	defaultTLSInterval = time.Second * 30
```

Comment:
As is, interval needs to be at least 1 minute, so `defaultTLSInterval` fails validation as is - bump to 1 min?

The 1 min minimum is arbitrarily picked, so can be changed if we see a pressing reason to do so.